### PR TITLE
docs:added guide for new users

### PR DESCRIPTION
**Pull request by tejaskh3**
Date: 26-01-2024

Developer Name: Tejas

Issue Ticket Number: #22 

### Description:
This documentation contains the guide for new developers, which can reduce the waiting time to get assigned with a task.

Backend Changes: No
Frontend Changes: No
Is Under Feature Flag: No
Database Changes: No

Breaking Changes: No

Deployment Notes: N/A 

Tested in Staging: Yes/No

### CODE REVIEW

**Review by iamitprakash — CHANGES_REQUESTED** [review]
can you add preview of md file

**Comment by tejaskh3** [comment]
> can you add preview of md file

For this, when you open my commit and start reading it this will show an option just click on that to view readme file.
attaching an image for reference and some steps.
1. click on "commit" above. 
2. Then open the file in which I have committed the changes and press the button in image.
image

And then anyone will be able to view Readme

File: docs/New User Guide/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Guide for New Users: Getting Assigned to Tasks
+
+Welcome! This document will guide new contributors on how to get assigned with a task in RDS projects.
+
+## Step 1: Choosing the task
+
+Now that you have completed your lift simulation, you are all set to contribute to the projects. There are two ways to get assigned with a task:
+
+1. **Using the Website or Set-Up Projects:**
```
Comment (satyam73) [inline docs/New User Guide/README.md]:
```suggestion
1. **Finding on you own:**

```

File: docs/New User Guide/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+3. **Add to Status Page:**
+
+   - Visit the [Status Page](https://status.realdevsquad.com/issues).
+   - Click on the 'Issues' tab and paste the GitHub issue title, then press 'Submit'.
+
+4. **Request Task:**
+
+   - On the Status Page, click 'Request as Task'.
+   - Set a reasonable time limit for task completion, the time you will need to get your code to production.
```
Comment (satyam73) [inline docs/New User Guide/README.md]:
```suggestion
   - Set a reasonable time limit for task completion, the time you will need to get your code to **production**. Once again, the time you entering here will be the time in which your code is in the **production** and ready to use(not any intermediary state).
```

File: docs/New User Guide/README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+   - After submitting, you will be redirected to a dashboard.
+     ![image](https://github.com/tejaskh3/uXHUB/assets/98630752/6d4069a5-1f00-44b6-a12a-b593400b0a9d)
+
+5. **Copy Dashboard URL and Tag for Approval:**
+
+   - Copy the dashboard URL.
+   - Tag Ankush, Tejas, or Prakash, requesting approval using channels `new-tasks` or `i-am-idle` for on discord.
```
Comment (satyam73) [inline docs/New User Guide/README.md]:
```suggestion
   - Tag Ankush, Tejas, or Prakash, requesting approval using channels `new-tasks` or `i-am-idle` on discord.

```

File: docs/New User Guide/README.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+   - And ask them to approve.
+
+     ![image](https://github.com/tejaskh3/uXHUB/assets/98630752/029a562c-fe8d-48ec-a16a-1991e69d7e23)
+
+By following these steps, you are all set to work on your assigned task. If you have questions, feel free to ask.
```
Comment (satyam73) [inline docs/New User Guide/README.md]:
```suggestion
By following these steps, you are all set to work on your assigned task. If you have questions, feel free to ask in the server's respective channel.

```

**Review by satyam73 — CHANGES_REQUESTED** [review]

**Review by satyam73 — APPROVED** [review]
LGTM

**Review by iamitprakash — APPROVED** [review]

**Pull request merged**